binder/Dockerfile

```diff
@@ -14,15 +14,4 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 ENV TAG="6b49f3337709"
 
-WORKDIR "${HOME}"
-COPY binder/README.ipynb .
-
-# Fix permissions on README.ipynb as root
-USER root
-
-RUN fix-permissions README.ipynb
-
-# Switch back to jovyan to avoid accidental container runs as root
-USER ${NB_UID}
-
-WORKDIR "${HOME}"
+COPY --chown=${NB_UID}:${NB_GID} binder/README.ipynb "${HOME}"/README.ipynb
```

Comment:
This actually makes the same as above but in a simple way.
Also, I don't think `fix-permissions` works on files, `--chown` does.